airflow/providers/amazon/aws/operators/kinesis_analytics.py

```diff
@@ -0,0 +1,348 @@
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Sequence
+
+from botocore.exceptions import ClientError
+
+from airflow.configuration import conf
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.kinesis_analytics import KinesisAnalyticsV2Hook
+from airflow.providers.amazon.aws.operators.base_aws import AwsBaseOperator
+from airflow.providers.amazon.aws.triggers.kinesis_analytics import (
+    KinesisAnalyticsV2ApplicationOperationCompleteTrigger,
+)
+from airflow.providers.amazon.aws.utils import validate_execute_complete_event
+from airflow.providers.amazon.aws.utils.mixins import aws_template_fields
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class KinesisAnalyticsV2CreateApplicationOperator(AwsBaseOperator[KinesisAnalyticsV2Hook]):
+    """
+    Creates an AWS Managed Service for Apache Flink application.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:KinesisAnalyticsV2CreateApplicationOperator`
+
+    :param application_name: The name of application. (templated)
+    :param run_time_environment: The runtime environment for the application. (templated)
+    :param service_execution_role: The IAM role used by the application to access services. (templated)
+    :param create_application_kwargs: Create application extra properties. (templated)
+    :param application_description: A summary description of the application. (templated)
+
+    :param aws_conn_id: The Airflow connection used for AWS credentials.
+        If this is ``None`` or empty then the default boto3 behaviour is used. If
+        running Airflow in a distributed manner and aws_conn_id is None or
+        empty, then default boto3 configuration would be used (and must be
+        maintained on each worker node).
+    :param region_name: AWS region_name. If not specified then the default boto3 behaviour is used.
+    :param verify: Whether to verify SSL certificates. See:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html
+    :param botocore_config: Configuration dictionary (key-values) for botocore client. See:
+        https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html
+    """
+
+    aws_hook_class = KinesisAnalyticsV2Hook
+    ui_color = "#44b5e2"
+
+    template_fields: Sequence[str] = aws_template_fields(
+        "application_name",
+        "run_time_environment",
+        "service_execution_role",
+        "create_application_kwargs",
+        "application_description",
+    )
+    template_fields_renderers: dict = {
+        "create_application_kwargs": "json",
+    }
+
+    def __init__(
+        self,
+        application_name: str,
+        run_time_environment: str,
```

Comment:
`runtime_environment` makes more sense?

```suggestion
        runtime_environment: str,
```